repo2docker/buildpacks/base.py

```diff
@@ -10,7 +10,7 @@
 import docker
 
 TEMPLATE = r"""
-FROM buildpack-deps:zesty
+FROM buildpack-deps:artful
```

Comment:
would it make sense to standardize on 16.04 LTS to lengthen the intervals of these types of updates?

Reply:
I think the current plan (from #148 and related issues) is to keep bumping until the next LTS release and then freeze there. I don't think we should go back several versions now.

Reply:
Ahh, that sounds even better! Thanks!

Reply:
Yes, I think once we get to 18.04, we shouldn’t use any non-LTS releases, but we should bump each release until then.